Dockerfile

```diff
@@ -29,7 +25,7 @@ COPY frontend/patches/* /headlamp/frontend/patches/
 WORKDIR /headlamp
 RUN cd ./frontend && npm install --only=prod
 
-FROM frontendinstall AS frontend
+FROM frontend-build AS frontend
```

Comment:
Nit: You removed the capitals in `AS`, but he it's remaining. I guess they should be consistent (IMO, capitals should be kept for keywords).